src/main/resources/index.jelly

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+Copyright 2022 CloudBees, Inc.
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<description>
+    This plugin provides support for iterating through all the <a href="https://javadoc.jenkins.io/jenkins/model/Nodes.html">Node</a>
+    instances that are in use by <code>Jenkins</code>, even those <code>Node</code> instances that are not traditionally attached to Jenkins. The API exposed by this
+    plugin can be used by cloud provider plugins to identify unused provisioned resource.
+  </description>
```

Comment:
Inconsistent indentation:

```suggestion
</description>
```

Also, per https://github.com/jenkinsci/jenkins/blob/44d0496f38584dcc9fffc42de4104b66753649a3/core/src/main/resources/hudson/PluginManager/installed.jelly#L110-L129 the description from `src/main/resources/index.jelly` is preferred over the `<description>` from `pom.xml` ("last resort" according to the Jenkins core source code), making the `<description>` from `pom.xml` now redundant. That is why https://github.com/jenkinsci/maven-hpi-plugin/blob/694a97af11fdc7b841ee220c17ed64dfaffe08b0/src/main/java/org/jenkinsci/maven/plugins/hpi/HpiMojo.java#L139-L143 says:

> Delete any `<description>` from `pom.xml` and create `src/main/resources/index.jelly`

This PR creates `src/main/resources/index.jelly`, but it does not delete the `<description>` from `pom.xml`.

Reply:
Description is a maven thing that has uses outside Jenkins, so I explicitly ignored that section.

Reply:
It does have theoretical uses outside of Jenkins, but I don't think there are any such uses in practice, and I think having the same information copied and pasted in multiple places slightly increases maintenance burden, so for that reason and consistency with existing precedent I still have a slight preference for deleting the `<description>` from `pom.xml`, but I don't feel strongly about this.

I do still think that the indentation should be made consistent by deleting the spaces, at the beginning of the line, but I wouldn't block this PR over some whitespace. I already approved the PR.

Reply:
the invalid html has been fixed `description` -> `div` along with the indent in 03e0f31